### ISSUE TITLE
Missing/incomplete A11y

### ISSUE THREAD

**Issue by cmumatt** [nanofuzz/nanofuzz#48]
We don't currently know how accessible the prototype tool is or what needs to be fixed given it was designed quickly for a short controlled experiment.
